### PR TITLE
fix(46666): No outlining span returned for multiline comment

### LINKED ISSUE #46666: No outlining span returned for multiline comment
# Bug Report

### 🔎 Search Terms

- getOutliningSpans
- fold / folding

### 🕗 Version & Regression Information
4.5.0-dev.20211029

### 💻 Repro

For the code:

```js
/*
x
*/

console.log(1) 
```

Try folding the multiline comment

### 🐛  Repro

No outlining span returned

One is returned if you remove `console.log(1)`

### PR DESCRIPTION
**Pull request by a-tarasyuk**
Fixes #46666

### CODE REVIEW

File: src/services/outliningElementsCollector.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ namespace ts.OutliningElementsCollector {
             if (depthRemaining === 0) return;
             cancellationToken.throwIfCancellationRequested();
 
-            if (isDeclaration(n) || isVariableStatement(n) || isReturnStatement(n) || n.kind === SyntaxKind.EndOfFileToken) {
+            if (isDeclaration(n) || isVariableStatement(n) || isReturnStatement(n) || isCallOrNewExpression(n) || n.kind === SyntaxKind.EndOfFileToken) {
```
Comment (andrewbranch) [inline src/services/outliningElementsCollector.ts:37]: If this is the fix, I don't understand why the bug only reproduced when the comment text had no leading spaces on its line. It feels like there must be something else at play as well?

Comment (a-tarasyuk) [reply src/services/outliningElementsCollector.ts:37]: Actually, I could not get an outline with space in the comment

Screenshot 2021-11-04 at 21 35 41
Screenshot 2021-11-04 at 21 35 48
Screenshot 2021-11-04 at 21 35 59

The comment associated with the call expression was not included in the outline spans.

Comment (andrewbranch) [reply src/services/outliningElementsCollector.ts:37]: I see—most be a difference with Monaco, because that’s the behavior I was seeing in the Playground.

**Review by andrewbranch — APPROVED** [review]

File: src/services/outliningElementsCollector.ts (hunk 1 above)
Comment (a-tarasyuk) [reply src/services/outliningElementsCollector.ts:37]: @andrewbranch Oke, thanks for the review 👍🏻

**Pull request merged**